src/app/backend-api.service.ts

```diff
@@ -1511,8 +1557,9 @@ export class BackendApiService {
         (fallback ?? this.GetDefaultProfilePictureURL(endpoint))
     );
   }
+
   GetDefaultProfilePictureURL(endpoint: string): string {
-    return this._makeRequestURL(endpoint, "/assets/img/default_profile_pic.png");
```

Comment:
renamed the old picture to have consistent name with all other images in the assets